packages/valory/skills/termination_abci/tests/test_behaviours.py

```diff
@@ -0,0 +1,547 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains round behaviours of FearAndGreedOracleAbciApp."""
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Callable, Dict, Hashable, List, Optional, Type
+from unittest import mock
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+
+from packages.valory.contracts.gnosis_safe.contract import GnosisSafeContract
+from packages.valory.contracts.multisend.contract import MultiSendContract
+from packages.valory.contracts.service_registry.contract import ServiceRegistryContract
+from packages.valory.protocols.contract_api import ContractApiMessage
+from packages.valory.protocols.contract_api.custom_types import RawTransaction, State
+from packages.valory.skills.abstract_round_abci.base import AbciAppDB
+from packages.valory.skills.abstract_round_abci.behaviour_utils import AsyncBehaviour
+from packages.valory.skills.abstract_round_abci.behaviours import BaseBehaviour
+from packages.valory.skills.abstract_round_abci.test_tools.base import (
+    FSMBehaviourBaseCase,
+)
+from packages.valory.skills.termination_abci.behaviours import (
+    BackgroundBehaviour,
+    TerminationAbciBehaviours,
+    TerminationBehaviour,
+)
+from packages.valory.skills.termination_abci.rounds import Event, SynchronizedData
+from packages.valory.skills.transaction_settlement_abci.behaviours import (
+    TransactionSettlementRoundBehaviour,
+)
+
+
+SERVICE_REGISTRY_ADDRESS = "0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA"
+SAFE_ADDRESS = "0x0"
+MULTISEND_ADDRESS = "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761"
+SERVICE_OWNER_ADDRESS = "0x0"
+SERVICE_ID = "termination_abci"
+
+
+@dataclass
+class BehaviourTestCase:
+    """BehaviourTestCase"""
+
+    name: str
+    initial_data: Dict[str, Hashable]
+    ok_reqs: List[Callable]
+    err_reqs: List[Callable]
+    expected_logs: List[str]
+
+
+class BaseTerminationTest(FSMBehaviourBaseCase):
+    """Base test case."""
+
+    path_to_skill = Path(__file__).parent.parent
```

Comment:
```suggestion
    path_to_skill = Path(__file__).parents(1)
```

Reply:
The pylint skip was actually copied over from another open-autonomy package, its probably a bit too much, and we can pin down the exact error we have to ignore.

Reply:
you have some pytest skip in other files as well that seems unnecessary